helper/config.go

```diff
@@ -1151,3 +1161,8 @@ func SetTestPrivPubKey(privKey secp256k1.PrivKey) {
 	}
 	pubKeyObject = pubKey
 }
+
+// SetTestInitialHeight sets test the initial height for testing
+func SetTestInitialHeight(height int64) {
+	initialHeight = height
+}
```

Comment:
<picky-mode>
I would be good to move this function to file `helper/config_test.go`, to avoid the risk of calling it in production by mistake.
</picky-mode>